packages/expo-calendar/src/Calendar.ts

```diff
@@ -1355,6 +1355,37 @@ export async function requestCalendarPermissionsAsync(): Promise<PermissionRespo
   return await ExpoCalendar.requestCalendarPermissionsAsync();
 }
 
+// @needsAudit
+/**
+ * Asks the user to grant write-only permissions for accessing and modifying user's calendars.
+ * This is a more limited scope than full calendar access, allowing only event creation and modification.
+ *
+ * @return A promise that resolves to an object of type [`PermissionResponse`](#permissionresponse).
+ * @platform ios
+ */
+export async function requestCalendarWritePermissionsAsync(): Promise<PermissionResponse> {
+  if (!ExpoCalendar.requestCalendarWritePermissionsAsync) {
+    throw new UnavailabilityError('Calendar', 'requestCalendarWritePermissionsAsync');
+  }
+  return await ExpoCalendar.requestCalendarWritePermissionsAsync();
+}
+
+// Also add a hook for write permissions
+/**
+ * Check or request write-only permissions to access the calendar.
+ * This is a more limited scope than full calendar access, allowing only event creation and modification.
+ *
+ * @example
+ * ```ts
+ * const [status, requestPermission] = Calendar.useCalendarWritePermissions();
+ * ```
+ * @platform ios
+ */
+export const useCalendarWritePermissions = createPermissionHook({
+  getMethod: getCalendarPermissionsAsync, // We'll still use the regular get method since it covers both
```

Comment:
why is the get method the same? write and read+write permissions should be different

Reply:
Oversight on my behalf, thanks